app/src/main/res/raw/credits_contributors.json

```diff
@@ -0,0 +1,347 @@
+[
+    {
+        "name": "Tobias Zwick",
+        "githubUsername": "westnordost",
+        "linesOfCodeChanged": 347675,
+        "linesOfInterfaceMarkupChanged": 130362,
+        "assetFilesChanged": 1637
+    },
+    {
+        "name": "Flo Edelmann",
+        "githubUsername": "FloEdelmann",
+        "linesOfCodeChanged": 15715,
+        "linesOfInterfaceMarkupChanged": 1645,
+        "assetFilesChanged": 112
+    },
+    {
+        "name": "Mateusz Konieczny",
+        "githubUsername": "matkoniecz",
+        "linesOfCodeChanged": 11777,
+        "linesOfInterfaceMarkupChanged": 2585,
+        "assetFilesChanged": 451
+    },
+    {
+        "name": "ENT8R",
+        "githubUsername": "ENT8R",
+        "linesOfCodeChanged": 7366,
+        "linesOfInterfaceMarkupChanged": 2246,
+        "assetFilesChanged": 399
+    },
+    {
+        "name": "Adrian Clay",
+        "githubUsername": "adrianclay",
+        "linesOfCodeChanged": 3920
+    },
+    {
+        "name": "Helium",
+        "githubUsername": "Helium314",
+        "linesOfCodeChanged": 3237,
+        "linesOfInterfaceMarkupChanged": 532,
+        "assetFilesChanged": 69
+    },
+    {
+        "name": "sumanabhi",
+        "githubUsername": "sumanabhi",
+        "linesOfCodeChanged": 2014,
+        "linesOfInterfaceMarkupChanged": 3159
+    },
+    {
+        "name": "Isira Seneviratne",
+        "githubUsername": "Isira-Seneviratne",
+        "linesOfCodeChanged": 1743
+    },
+    {
+        "name": "Adrien Pavie",
+        "githubUsername": "PanierAvide",
+        "linesOfCodeChanged": 1639,
+        "linesOfInterfaceMarkupChanged": 262,
+        "assetFilesChanged": 48
+    },
+    {
+        "name": "Matija Nalis",
+        "linesOfCodeChanged": 1493,
+        "assetFilesChanged": 22
+    },
```

Comment:
Do you know why it doesn't add [@mnalis](https://github.com/mnalis)'s GitHub user name?

Reply:
No

Reply:
Because the GitHub API does not specify the GitHub account associated with his latest contribution:
https://github.com/streetcomplete/StreetComplete/commit/2efd117a5ef1321716108e61f890c58b85273e64

Also on the website, this is missing. (E.g. I cannot click on his name.)

Reply:
That's most likely because he's using an email that is not connected to his GitHub account.

Reply:
Interesting. @mnalis do you know what's going on? Maybe you'd have to specify the email used in that commit (see https://github.com/streetcomplete/StreetComplete/commit/2efd117a5ef1321716108e61f890c58b85273e64.patch) somewhere in your GitHub settings? Or is this deliberate?

Anyway, this is not merge-blocking for this PR.

Reply:
added, does it work now?

Reply:
Yes, it works now! However, I don't think it's worth an extra commit now, given that this task will be run before the next release anyway.

Reply:
For the following contributors, the same applies: Please specify your email address (that you used in Git; see the linked commit patches) as a private email in your GitHub account settings. Otherwise, your contributions might be counted as separate accounts, and thus might not pass the significance threshold to be displayed (or at least won't be linked to your GitHub profile).

* @krzyk: https://github.com/streetcomplete/StreetComplete/commit/e991b0b7d1769e184792225ad127a36f7f5ed1f1.patch
* @fatal69100: https://github.com/streetcomplete/StreetComplete/commit/ca6d0c3271939c35cc6edba6a69e5fa7eb8be093.patch
* @ka7: https://github.com/streetcomplete/StreetComplete/commit/0c2c315ad73891914388f08b3ca3be0ac5655d1f.patch
* (unknown): https://github.com/streetcomplete/StreetComplete/commit/571d33cd490662824b2ed79b1c6f98c12a6c6859.patch
* @cyclingcat: https://github.com/streetcomplete/StreetComplete/commit/b7e108f9e6c23fce5cb9608424a46a87e7463748.patch
* @Atrate: https://github.com/streetcomplete/StreetComplete/commit/dd5bfb960a8183f29d166cf48033ad3870152426.patch
* @gileri: https://github.com/streetcomplete/StreetComplete/commit/d3556d2651720ba941d0f505385e78d2e373d47b.patch